### PR TITLE
Create experiment only if it's not exist.

### PR DESCRIPTION
**Pull request by hongye-sun**
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/pipelines/892)

### CODE REVIEW

File: sdk/python/kfp/_client.py [hunk 1]
```diff
@@ -108,7 +108,7 @@ def create_experiment(self, name):
     """
     import kfp_experiment
 
-    experiment = self.get_experiment(experiment_name=name)
+    experiment = self.get_experiment(experiment_name=name, ignore_error=True)
```
Comment (Ark-kun) [inline sdk/python/kfp/_client.py]: AFAIK, the pythonic way to accomplish this is to just use `try`/`except`.

Comment (hongye-sun) [reply sdk/python/kfp/_client.py]: You are right. Done.

**Pull request merged**